ably/export_test.go

```diff
@@ -103,6 +103,12 @@ func (os ClientOptions) RealtimeRequestTimeout(d time.Duration) ClientOptions {
 	})
 }
 
+func (os ClientOptions) ConnectionStateTTL(d time.Duration) ClientOptions {
```

Comment:
PS. Now that I think of it, you won't have a ConnectionDetails given RTN14 is about establishing a connection. It really doesn't make sense that [RTN14e](https://docs.ably.io/client-lib-development-guide/features/#RTN14e) is under RTN14, because it seems to be about recovering an existing connection. Connections not yet established don't have a state! I'll bring this up.

Reply:
@gernest The situation here has been clarified in https://github.com/ably/docs/pull/974. Ignore my previous comments; RTN14e and SUSPENDED is _not_ (only) about connection state recovery. So it makes sense it can be set also as an option for tests.

(Although I think it's preferable to mock the timeout code to control it from tests, instead of running real timeouts with artificially small durations.)